v23.1/advanced-changefeed-configuration.md

```diff
@@ -9,37 +9,140 @@ docs_area: stream_data
 The configurations and settings explained on this page will significantly impact a changefeed's behavior.
 {{site.data.alerts.end}}
 
-The following sections describe settings, configurations, and details to tune changefeeds for these use cases:
+The following sections describe performance, settings, configurations, and details to tune [changefeeds](change-data-capture-overview.html):
 
+- [Changefeed performance](#changefeed-performance)
 - [High durability delivery](#tuning-for-high-durability-delivery)
 - [High throughput](#tuning-for-high-throughput)
 
-Some specific options for the `kafka_sink_config` and `webhook_sink_config` parameters are discussed on this page. However, for more information on more specific tuning for Kafka and Webhook sinks see the following pages:
+Some options for the `kafka_sink_config` and `webhook_sink_config` parameters are discussed on this page. However, for more information on specific tuning for Kafka and Webhook sinks, refer to the following pages:
 
 - [Kafka sinks](changefeed-sinks.html#kafka-sink-configuration)
 - [Webhook sinks](changefeed-sinks.html#webhook-sink-configuration)
 
+## Changefeed performance
+
+{% include_cached new-in.html version="v23.1" %} By default, changefeeds are integrated with elastic CPU, which helps to prevent changefeeds from affecting foreground traffic. For example, [changefeed backfills](changefeed-messages.html#schema-changes-with-column-backfill) and [initial scans](create-changefeed.html#initial-scan) can be CPU-intensive. This integration will result in a cluster prioritizing SQL traffic over changefeeds. Since this may affect changefeed latency, you can monitor your cluster's [admission control system](admission-control.html) on the [Overload Dashboard](ui-overload-dashboard.html) and changefeed latency on the [Changefeed Dashboard](ui-cdc-dashboard.html).
+
+This is controlled by the following [cluster settings](cluster-settings.html), which are by default enabled:
```

Comment:
How do we feel about listing these cluster settings? Should we remove from public docs?